faculty-affiliations.csv

```diff
@@ -774,14 +774,34 @@ Panayotis Mavromatis , New York University
 Patrick Cousot , New York University
 Ralph Grishman , New York University
 Richard Bonneau , New York University
-Richard Cole 0001, New York University
+Richard Cole , New York University
```

Comment:
This is incorrect: AFAICT the correct DBLP entry for Richard Cole is in fact Richard Cole 0001 (see http://dblp.uni-trier.de/pers/hd/c/Cole_0001:Richard).